### PR TITLE
feat: move towards arcus-style project

### PR DESCRIPTION
**Pull request by stijnmoreels**
FIrst change towards Arcus-style project.

### CODE REVIEW

**Comment by stijnmoreels** [comment]
If I read the license correctly, we should add a notice on every file we change?

**Comment by tomkerkhove** [comment]
> If I read the license correctly, we should add a notice on every file we change?

Let's do it to make sure as well as in the README

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Azure Function Health
+# Arcus - Web API Azure Function Health
```
Comment (tomkerkhove) [inline README.md:2]: We should add that this is a fork of https://github.com/keithsmith21/AzureFunctionHealth and why we are forking it (extend and offer as NuGet package)

Comment (tomkerkhove) [reply README.md:2]: We should clean the README up once we have aligned it to remove outdated things

**Review by tomkerkhove — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Arcus - Web API Azure Function Health
 
+> Copyright &copy; Apache License 2.0 2021 @keithsmith21 All rights reserved
+> > Forked from https://github.com/keithsmith21/AzureFunctionHealth
+> >
+> > State changes: project files were renamed, samples were removed, documentation was reprhased and moved to dedicated site.
```
Comment (tomkerkhove) [inline README.md:6]: We don't have to list what we changed but more why we have forked it

File: README.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Arcus - Web API Azure Function Health
 
+> Copyright &copy; Apache License 2.0 2021 @keithsmith21 All rights reserved
+> > Forked from https://github.com/keithsmith21/AzureFunctionHealth
```
Comment (tomkerkhove) [inline README.md]:
```suggestion
> This is a fork of https://github.com/keithsmith21/AzureFunctionHealth, all rights reserved.
```

File: README.md (hunk 2 above)
Comment (stijnmoreels) [reply README.md:6]: No? As it states this in the license that we should add a copyright notice on every file plus the things changed. Or I'm I reading that wrong?

Comment (tomkerkhove) [reply README.md:6]: Yes, but this is the readme so it's ok to use a different approach here imo

Comment (stijnmoreels) [reply README.md:6]: Ah, ok. Makes sense.

File: README.md (hunk 1 above)
Comment (stijnmoreels) [reply README.md:2]: Tracked here #2 , resolving.

**Pull request merged**